load-test-framework/src/main/resources/command.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+package pubsub;
+
+option java_package = "com.google.pubsub.flic.common";
+option java_outer_classname = "Command";
+
+service LoadtestFramework {
+  rpc StartClient(CommandRequest) returns (google.protobuf.Empty) {
+  }
+}
+
+message CommandRequest {
```

Comment:
The tasks will be different clients. So the instance started will already be a publisher / subscriber, and these are the arguments that will be given. (I.e. there will be startup scripts that provide initial arguments, like subscribe, publish, etc. that are defining of the type of client. We could put that in the command instead of the script, there's no material difference, but it seems easier since then each instance group / template can be separated by publish/subscribe and type. We should loop up tomorrow to discuss further. This is definitely not the full list of options, but it was my first pass.